wger/nutrition/migrations/0002_auto_20180524_1451.py

```diff
@@ -1,30 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Why have you deleted this migration file?

Reply:
i was having a challenge making migrations, since i added a new field for time, to the meal_item model. I kept getting a message no new migrations to apply but the field hadn't been added to the table.